support-frontend/assets/pages/aus-moment-map/ausMomentMap.tsx

```diff
@@ -145,7 +160,12 @@ const AusMomentMap = () => {
 					{windowWidthIsLessThan('desktop') && <Blurb />}
 					<Map
 						selectedTerritory={selectedTerritory}
-						setSelectedTerritory={(territory) => {
+						setSelectedTerritory={(
+							territory:
+								| string
+								| ((prevState: string | null) => string | null)
+								| null,
+						) => {
```

Comment:
I think from the `Map` components perspective it should only need a function that's `(territory: string) => void`